sentinel/tests/unit/lib/metadata.js

```diff
@@ -0,0 +1,103 @@
+const sinon = require('sinon');
+const chai = require('chai');
+const db = require('../../../src/db');
+const metadata = require('../../../src/lib/metadata');
+
+describe('metadata', () => {
+
+  afterEach(() => sinon.restore());
+
+  describe('getProcessedSeq', () => {
+
+    it('handles missing metadata doc', () => {
+      sinon.stub(db.sentinel, 'get').rejects({ status: 404 });
+      sinon.stub(db.medic, 'get').rejects({ status: 404 });
+      return metadata.getProcessedSeq(seq => {
```

Comment:
I think this is supposed to be: `return metadata.getProcessedSeq().then(seq => {`. 
There are a few more tests that have the same issue.

Reply:
Yup, these tests weren't doing anything. Great spotting.